src/py42/clients/detectionlists/high_risk_employee.py

```diff
@@ -0,0 +1,150 @@
+import json
+
+from py42.clients import BaseClient
+
+
+class HighRiskEmployeeClient(BaseClient):
+    """
+    TODO Documentation here
+    """
+
+    # In other modules URI is a single variable, breaking it up to have better control
+    # if we want to refactor version and prefix to a common configuration.
+    _api_version = u"v2"
+    _uri_prefix = u"/svc/api/{0}".format(_api_version)
+
+    def __init__(self, session, user_context):
+        super(HighRiskEmployeeClient, self).__init__(session)
+        self._user_context = user_context
+
+    def add(self, user_id):
```

Comment:
@alanag13 
Here, we don't create user, `/user/create/` endpoint, directly we add `/highriskemployee/add`. I removed that part after change in requirement, Is this ok? From my understanding and testing I think, user existence is a pre-requisite for adding.

Difference in code can be seen between commits, in `create` method v/s this method.

Reply:
Changed to allow only user_id.